plugins/python-build/share/python-build/micropython-1.9.3

```diff
@@ -0,0 +1,2 @@
+#require_gcc
+install_git micropython-1.9.3 https://github.com/micropython/micropython "tag v1.9.3" micropython
```

Comment:
For this specific case, what we need to do is not checking out git tag, downloading archive of the tagged release, in my opinion. We can download the archive of v1.9.3 from https://github.com/micropython/micropython/releases/download/v1.9.3/micropython-1.9.3.tar.gz